networkx/generators/geometric.py

```diff
@@ -784,3 +785,110 @@ def thresholded_random_geometric_graph(
     )
     G.add_edges_from(edges)
     return G
+
+
+@py_random_state(5)
+def S1_graph(n, beta, gamma, mean_degree, kappas=None, seed=None):
+    r"""Returns a $\mathbb{S}^1$ model.
+
+    The $\mathbb{S}^1$ model is the simplest among the class of geometric models [1].
+    The similarity space is a one dimensional sphere—a circle of radius R—where N nodes are
+    distributed with a fixed density, set to one without loss of generality, so that $N = 2\pi R$.
+    Each node is also given a hidden variable ``'kappa'`` proportional to its expected degree.
```

Comment:
```suggestion
    Each node is also given a hidden variable ``kappa`` proportional to its expected degree.
```

Just some cosmetic changes here, we already have `kappa` in a code block no need to add it inside quotes :)
Could you also do remove these quotes from all the other codeblocks too? Thanks!

You can check how this doc is rendered here https://output.circle-artifacts.com/output/job/6af83fc3-9162-4cab-9fab-75baa39c2fd9/artifacts/0/doc/build/html/reference/generated/networkx.generators.geometric.S1_graph.html#s1-graph

Reply:
Sure! I removed the quotes